english/the-slayer/freddie/freddie.md

```diff
@@ -1,2 +1,12 @@
-Freddie saved the little marshmallows stabing the terrible marshmallow with
-his chocolate stick.
\ No newline at end of file
+Freddie saved the little marshmallows stabing the terrible marshmallow with his chocolate stick.
+
+As the terrible marshmallow lay heaped on the floor, he wheezed a final threat at Freddie, 
+"You think you've won.  You think you've defeated me.  But there s'more."  
+With that the terrible marshmallow began to convulse in a mix of evil laughing and gasps for air.
+
+Freddie stood over the terrible marshmallow, trying to think of a witty retort.
+Nothing was coming to his head.  There was only one thing left to do.
+
+[Practice yoga.](english/yoga/breathe/breathe.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.